gap/congruences/cong.gi

```diff
@@ -494,3 +494,18 @@ _GenericCongCanonicalLookup);
 
 MakeReadWriteGlobal("_GenericCongCanonicalLookup");
 Unbind(_GenericCongCanonicalLookup);
+
+InstallMethod(EquivalenceRelationCanonicalPartition,
```

Comment:
Similar comment as before. This method is exactly the same as for Left/RightSemigroupCongruences. In my opinion, therefore this method can be deleted. Since a (two-sided) congruence is both a left and a right congruence, one of those (identical) methods will be selected instead. You could add a comment line stating that for two-sided congruences, the left/right versions apply.